src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java

```diff
@@ -94,32 +108,33 @@ public OAuth2AccessToken readAccessToken(String token) {
     /**
      * Revokes the access tokens of the user with the given ID.
      *
-     * @param id
-     *            the user ID
-     * @param token
-     *            the token to access the service
+     * @param id    the user ID
+     * @param token the token to access the service
      */
     public void revokeAccessTokens(String id, String token) {
         AccessToken accessToken = new AccessToken.Builder(token).build();
-        OsiamConnector connector = createConnector();
-        connector.revokeAllAccessTokens(id, accessToken);
+        try {
+            connector.revokeAllAccessTokens(id, accessToken);
+        } catch (ConnectionInitializationException e) {
+            String message = "Unable to revoke access token on auth-server " + authServerHome + ": " + e.getMessage();
+            LOGGER.warn(message);
```

Comment:
Please use `LOGGER.warn("Unable to revoke access token on auth-server {}: {}", authServerHome, e.getMessage());`. Thanks @sputnik27 for the hint ;)